package.json

```diff
@@ -54,6 +54,7 @@
     "@types/react-router-dom": "^5.1.6",
     "bundlesize": "^0.18.0",
     "eslint-config-bliss": "^5.0.0",
+    "eslint-plugin-cypress": "^2.11.2",
```

Comment:
This is one of the merge conflicts that appears.

```
<<<<<<< HEAD
    "eslint-plugin-cypress": "^2.11.2",
=======
    "eslint-config-react-app": "^6.0.0",
>>>>>>> master
```

Do we want to keep both changes, adding `"eslint-plugin-cypress": "^2.11.2",` from this branch, `feat/paginate-search-results`, and `"eslint-config-react-app": "^6.0.0",`, from the main branch?

Reply:
yea both changes should be kepth

Reply:
Okay, thanks.